chart/hyrax/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: hyrax
 description: An open-source, Samvera-powered digital repository system
 type: application
-version: 1.3.0
+version: 1.3.1
```

Comment:
if we're being fussy about semver i think this counts as adding functionality so it might want to be a minor version bump

Reply:
oh that's true, thanks for that. I'll switch to `1.4.0`